src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -366,7 +426,7 @@ public void togglePermission(Rank rank, @NotNull Action action)
      * @param rank Desired rank
      * @return True if successful, otherwise false
      */
-    public boolean setPlayerRank(UUID id, Rank rank)
+    public boolean setPlayerRank(UUID id, Rank rank, World world)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document the parameter(s): world [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUndocumentedApi)